tests/splinterdb_test_apis.h

```diff
@@ -17,8 +17,8 @@
 void
 splinterdb_cache_flush(const splinterdb *kvs);
 
-void *
-splinterdb_get_heap_handle(const splinterdb *kvs);
+platform_heap_id
+splinterdb_get_heap_id(const splinterdb *kvs);
```

Comment:
As `heap_handle` has been obsoleted as part of previous commit, rework this interface to now return `heap_id`.